deepfence_agent/tools/apache/scope/probe/kubernetes/service.go

```diff
@@ -75,7 +75,7 @@ func (s *service) GetNode() report.TopologyNode {
 	return report.TopologyNode{
 		Metadata: metadata,
 		Parents: &report.Parent{
-			CloudProvider:     cloudProviderNodeId,
+			//CloudProvider:     cloudProviderNodeId,
```

Comment:
Why are we removing this one?

Reply:
I don't see it being used in the console.
It is not a global variable anymore.
We can add if required in future.